Formula/n/newsraft.rb

```diff
@@ -0,0 +1,28 @@
+class Newsraft < Formula
+  desc "Terminal feed reader"
+  homepage "https://codeberg.org/newsraft/newsraft"
+  url "https://codeberg.org/newsraft/newsraft/archive/newsraft-0.29.tar.gz"
+  sha256 "1dbec925d5556cae70e92dcc7b69cc4899473deb1d26a600b8903dbc0b3fde65"
+  license "ISC"
+
+  depends_on "scdoc" => :build
+  depends_on "gumbo-parser"
```

Comment:
This repo is archived and the formula should be deprecated. I don't think we should add new software that depends on it

Reply:
@SMillerDev perhaps it needs to be updated, but current release works just fine.

Here's the repository: https://codeberg.org/gumbo-parser/gumbo-parser

Reply:
yeah, [old repo was indeed archived](https://github.com/Homebrew/homebrew-core/pull/172148), https://github.com/google/gumbo-parser, but since then we [also switched to the codeberg one](https://github.com/Homebrew/homebrew-core/pull/183286)